content/en/docs/installation/supported-releases.md

```diff
@@ -0,0 +1,262 @@
+
+| Release | Release Date |        EOL        | [Supported Kubernetes versions][s] |
+| ------- | :----------: | :---------------: | :--------------------------------: |
+| [1.3][] | Apr 08, 2021 |   [~][]Aug 2021   | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+| [1.2][] | Feb 10, 2021 | [~][]Jun 11, 2021 | 1.16, 1.17, 1.18, 1.19, 1.20, 1.21 |
+
+## Upcoming Releases
+
+| Release |   Release Date    |      EOL      | [Supported Kubernetes versions][s] |
+| ------- | :---------------: | :-----------: | :--------------------------------: |
+| [1.4][] | [~][]Jun 11, 2021 | [~][]Oct 2021 |                TBC                 |
+
+> The `~` sign is used when the date is uncertain and might change; the
+> "EOL" abbreviation stands for End Of Life.
+
+## Old Releases
+
+| Release  | Release Date |     EOL      | Compatible Kubernetes versions |
+| -------- | :----------: | :----------: | :----------------------------: |
+| [1.1][]  | Nov 24, 2021 | Apr 08, 2021 |          1.11 → 1.21           |
+| [1.0][]  | Sep 02, 2020 | Feb 10, 2021 |          1.11 → 1.21           |
+| [0.16][] | Jul 23, 2020 | Nov 24, 2020 |          1.11 → 1.21           |
+| [0.15][] | May 06, 2020 | Sep 02, 2020 |          1.11 → 1.21           |
+| [0.14][] | Mar 11, 2020 | Jul 23, 2020 |          1.11 → 1.21           |
+| [0.13][] | Jan 21, 2020 | May 06, 2020 |          1.11 → 1.21           |
+| [0.12][] | Nov 27, 2019 | Mar 11, 2020 |          1.11 → 1.21           |
+| [0.11][] | Oct 10, 2019 | Jan 21, 2020 |           1.9 → 1.21           |
+
+[s]: #kubernetes-supported-versions
+[~]: https://docs.google.com/document/d/1Tc5t6ylY9dhXAan1OjOoldeaoys1Yh4Ir710ATfBa5U/edit?pli=1#bookmark=id.jzi02xg0ngn "Project timeline"
+[1.4]: https://github.com/jetstack/cert-manager/milestone/25
+[1.3]: https://cert-manager.io/docs/release-notes/release-notes-1.3
+[1.2]: https://cert-manager.io/docs/release-notes/release-notes-1.2
+[1.1]: https://cert-manager.io/docs/release-notes/release-notes-1.1
+[1.0]: https://cert-manager.io/docs/release-notes/release-notes-1.0
+[0.16]: https://cert-manager.io/docs/release-notes/release-notes-0.16
+[0.15]: https://cert-manager.io/docs/release-notes/release-notes-0.15
+[0.14]: https://cert-manager.io/docs/release-notes/release-notes-0.14
+[0.13]: https://cert-manager.io/docs/release-notes/release-notes-0.13
+[0.12]: https://cert-manager.io/docs/release-notes/release-notes-0.12
+[0.11]: https://cert-manager.io/docs/release-notes/release-notes-0.11
+
+You can find available releases on the [releases
+page](https://github.com/cert-manager/cert-manager/releases). You can find
+the release notes for each minor release
+[here](https://cert-manager.io/docs/release-notes/), and the upgrade
+instructions are
+[here](https://cert-manager.io/docs/installation/upgrading/). The
+cert-manager release process is documented on the [release-process
+page](https://cert-manager.io/docs/contributing/release-process/).
+
+## Support policy
+
+### What we mean by support
```

Comment:
nit: title case 
```suggestion
### What We Mean by Support
```

Reply:
Fixed in [e05d5d2](https://github.com/cert-manager/website/commit/e05d5d245379f4ebc7ac4fd7fe25a4265437c4c7) (consisntent casing)